Gemfile

```
@@ -10,10 +10,14 @@ group :development, :test do
   gem "draper"
   gem "devise"
 
-  gem "rails", "~> 7.0.0"
+  gem "rails", "~> 7.1.0"
 
   gem "sprockets-rails"
   gem "sassc-rails"
+  # Wait until next version (current v4.0.0) is released. (We need https://github.com/activerecord-hackery/ransack/pull/1439 released).
+  # Then remove this line
+  gem "ransack", github: "activerecord-hackery/ransack", branch: "main"
+  gem "formtastic", github: "lsylvester/formtastic", branch: "rails-71-support"
```

For completeness with the others, this can be switched to the default branch now.

```suggestion
  gem "formtastic", github: "formtastic/formtastic", branch: "master"
```

---

Thanks! You are right. I forgot to update this on my latest push. Done